recipes/gj2ascii/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "0.4.1" %}
+
+package:
+  name: gj2ascii
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/g/gj2ascii/gj2ascii-{{ version }}.tar.gz
+  sha256: 65c865410aecd6e4698a120b47531171ee4bc89a511c301c770393f4884ffe8e
+
+build:
+  number: 0
+  # FXIME: retry when the entry_points issue is resolved.
```

Comment:
Is there another issue with `entry_points` and  `noarch: python` than https://github.com/conda/conda-build/issues/2362? (That seems like it shouldn't be a problem here.)

Reply:
I get here the same error as https://github.com/conda-forge/cc-plugin-glider-feedstock/pull/7

Not a syntax error, a file not found. I did not test latest `conda-build` but I believe that @msarahan may have already solved it.

Reply:
Huh, okay. Different from the failure I saw before. :man_shrugging: